k8s/namespaces/wa/wa-task-batch-termination-failure/aat-00.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: wa-task-batch-termination-failure
+spec:
+  values:
+    job:
+      schedule: "0-59/10 * * * *"
```

Comment:
You probably want a different schedule within each cluster otherwise they'll run at the same time giving you the same result.